helm/monitoring/templates/grafana/service.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    app: grafana
+  ports:
+    - port: {{ .Values.grafana.service.port }}
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)